### DISCUSSION TITLE
Visualizing Named Sets in Paraview

### DISCUSSION THREAD

**Discussion by vikramroybarc** [idaholab/moose#28079] [Q&A Tools]
Hi, 

I recently started learning moose. I have some doubts regarding visualizng named sets of mesh in PARAVIEW.

In this example (https://github.com/idaholab/moose/blob/next/modules/solid_mechanics/examples/bridge/bridge.i) different boundaries are named top, bottom1, ... based on boundary_id variable.

Similarly in the piston example (https://github.com/idaholab/moose/blob/next/modules/solid_mechanics/examples/piston/piston_params.i) load is applied on the load_surf boundary

I understand that these boundary related info (boundary_id in bridge example and named_set-> load_surf in piston example) variable is written in exodus mesh file. How do I visualize these named sets in paraview.

Anybody having idea regarding this kindly help.

Thankyou

**Comment by GiudGiud** [comment]
Hello

On the info tab on the left of the screen in Paraview. Sidesets and nodesets will appear there with their names

<img width="344" alt="Screenshot 2024-07-08 at 7 47 30 AM" src="https://github.com/idaholab/moose/assets/25037151/ebbe2da4-9489-4f68-a8c2-b40d0e12332f">

you may have to click the cog icon to turn on advanced mode.

**Comment by vikramroybarc** [comment]
Hi Giud,

Thanks for your reply.

I am using Paraview 5.9.0. In this version of paraview, I can see some sidesets are defined in the info tab as seen in this image.

![Screenshot from 2024-07-08 23-26-46](https://github.com/idaholab/moose/assets/126593761/044de8cf-4161-4edc-9869-5256f7199039)

But How can I visualize which region of mesh each of these sideset corresponds to.

Kindly help if you have an idea regarding this

**Reply by GiudGiud** [reply]
Instead of the info tab, use the properties tab (all the way to the left-center of your screenshot)

then select the sidesets in the menu similar to my screenshot

**Comment by vikramroybarc** [comment]
Thankyou very Guid for the help. I am finally able to do this.
